packages/renderer/src/lib/extensions/ExtensionDetailsLink.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa';
+import { router } from 'tinro';
+
+import Tooltip from '/@/lib/ui/Tooltip.svelte';
+import type { CombinedExtensionInfoUI } from '/@/stores/all-installed-extensions';
+
+export let extension: CombinedExtensionInfoUI;
+
+export let displayIcon: boolean = true;
+
+function openDetailsExtension() {
+  router.goto(`/extensions/details/${extension.id}/`);
+}
+</script>
+
+<Tooltip tip="{extension.name} extension details" top>
+  <button aria-label="{extension.name} extension details" type="button" on:click="{() => openDetailsExtension()}">
+    <div class=" flex flex-row items-baseline">
```

Comment:
```suggestion
    <div class="flex flex-row items-baseline items-center">
```

| Before | After |
| --- | --- |
| ![image](https://github.com/containers/podman-desktop/assets/42176370/d7534806-5ec5-4abf-a4ba-68da27d370f1) | ![image](https://github.com/containers/podman-desktop/assets/42176370/5ad003e8-b08a-4af0-9875-cd4fef57562e)|

Reply:
I removed items-baseline then